### PR TITLE
odies.net added

### PR DESCRIPTION
**Pull request by brisdevs**

### CODE REVIEW

**Comment by brisdevs** [comment]
The plug-in works smoothly and stably. All errors are fixed

File: websites/O/Odies/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    },
+
+    "service": "Odies",
+    "description": {
+      "tr": "Odies, botlarınız ve siteleriniz için ücretsiz kod, yardım alabileceğiniz bir sitedir.",
+      "en": "Odies is a free code for your bots and sites, a site where you can get help."
+    },
+
+    "url": "odies.net",
+    "version": "1.0.7",
```
Comment (Bas950) [inline websites/O/Odies/dist/metadata.json:15]: v 1.0.0 on release

**Review by Bas950 — CHANGES_REQUESTED** [review]
Remove strange spacings in the metadata, upload proof of it working

File: websites/O/Odies/dist/metadata.json (hunk 1 above)
Comment (brisdevs) [reply websites/O/Odies/dist/metadata.json]: **Too many blank lines at the end of the file fixed & Error page added &  Version arrangement has been made**

**Comment by ririxidev** [comment]
Sorry but, domain is not 2 months old. It's 1 month and 9 days old. Sorry, but we only accepted domains with age min. 2 months.

**Pull request closed without merging**